egs/vivos/asr1/RESULTS.md

```diff
@@ -1,120 +1,487 @@
-# pytorch Transducer (enc: VGG2L + 6 x Transformer, dec: 1 x LSTM)
+# RNN-Transducer (enc: 4 x BLSTMP, dec: 1 x LSTM)
 
 - Environments
-  - date: `Fri Apr  3 10:34:40 CEST 2020`
+  - date: `Sat Aug 22 16:14:35 CEST 2020`
   - python version: `3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]`
   - espnet version: `espnet 0.6.2`
   - chainer version: `chainer 6.0.0`
   - pytorch version: `pytorch 1.0.1.post2`
-  - Git hash: `d19105b0793b1a9c6e11293f2f95f85699632367`
-  - Commit date: `Fri Apr 3 10:09:03 2020 +0200`
-
-- Model files (archived to model.tar.gz by `$ pack_model.sh`)
-  - model link: https://drive.google.com/file/d/1mW2ANumwGozam7mt2_0JSpUAWIFfvyac
-  - training config file: `conf/tuning/transducer/train_transformer-rnn_transducer.yaml`
-  - decoding config file: `conf/tuning/transducer/decode_transducer.yaml`
-  - cmvn file: `data/train_nodev/cmvn.ark`
-  - e2e file: `exp/train_nodev_pytorch_train_transformer-rnn_transducer/results/model.last5.avg.best`
-  - e2e JSON file: `exp/train_nodev_pytorch_train_transformer-rnn_transducer/results/model.json`
-  - lm file: `exp/train_rnnlm_pytorch_lm_word7184/rnnlm.model.best`
-  - lm JSON file: `exp/train_rnnlm_pytorch_lm/model.json`
-  - dict file: `data/lang_1char`
-
-- Results (obtained with `$ show_results.sh`)
+  - Git hash: `077c8970afb477e059932f7243a9b728c8ab1a69`
+  - Commit date: `Fri Aug 21 16:32:27 2020 +0200`
 
 ## CER
 
 |dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
 |---|---|---|---|---|---|---|---|---|
-|decode_test_decode_transducer|760|32771|85.4|10.8|3.8|3.0|17.6|94.2|
-|decode_test_decode_transducer_lm|760|32771|85.6|10.6|3.8|3.0|17.4|93.3|
-|decode_test_decode_transducer_lm_word7184|760|32771|86.1|10.2|3.7|2.8|16.7|92.1|
-|decode_train_dev_decode_transducer|100|4007|84.7|12.8|2.6|3.2|18.6|97.0|
-|decode_train_dev_decode_transducer_lm|100|4007|84.7|12.6|2.7|3.3|18.6|97.0|
-|decode_train_dev_decode_transducer_lm_word7184|100|4007|84.9|12.4|2.7|3.3|18.4|97.0|
+|decode_test_decode_alsd|760|32771|85.3|11.5|3.2|3.6|18.3|93.8|
+|decode_test_decode_alsd_lm|760|32771|85.7|11.1|3.2|3.6|17.9|93.0|
+|decode_test_decode_alsd_lm_word7184|760|32771|86.2|10.7|3.1|3.5|17.3|92.0|
```

Comment:
What is difference between lm and lm_word7184?
I think it's enough to keep the best result about lm.

Reply:
`lm` is a char LM and the other a word LM, I just followed the style from previous CTC results! I can remove one of them but the best result can be different in accordance to CER or WER.

Reply:
Word LM looks also better for CER ?

Reply:
There are less cases now than I thought, I count only three cases where CER with word lm is greater than with char lm. I'll delete entries with char lm.